src/python/pants/backend/pants_info/list_target_types.py

```diff
@@ -25,6 +25,7 @@
     StringOrStringSequenceField,
     StringSequenceField,
     Target,
+    TriBoolField,
```

Comment:
Love it. Great idea.

Reply:
I remember we used to (maybe still do?) have a specific option type for this so I wanted to do the same in code for targets!